packages/antd/src/widgets/CheckboxWidget/index.tsx

```diff
@@ -0,0 +1,54 @@
+import React from "react";
+import { WidgetProps } from "@rjsf/utils";
+import Checkbox, { CheckboxChangeEvent } from "antd/lib/checkbox";
+
+const CheckboxWidget = ({
+  autofocus,
+  disabled,
+  formContext,
+  id,
+  label,
+  onBlur,
+  onChange,
+  onFocus,
+  // options,
+  // placeholder,
+  readonly,
+  // required,
+  // schema,
+  value,
+}: WidgetProps) => {
+  const { readonlyAsDisabled = true } = formContext;
+
+  const handleChange = ({ target }: CheckboxChangeEvent) =>
+    onChange(target.checked);
+
+  const handleBlur = ({ target }: React.FocusEvent<HTMLInputElement>) =>
+    onBlur(id, target.checked);
+
+  const handleFocus = ({ target }: React.FocusEvent<HTMLInputElement>) =>
+    onFocus(id, target.checked);
+
+  // Antd's typescript definitions do not contain the following props that are actually necessary and, if provided,
+  // they are used, so hacking them in via by spreading `extraProps` on the component to avoid typescript errors
+  const extraProps = {
+    onBlur: !readonly ? handleBlur : undefined,
+    onFocus: !readonly ? handleFocus : undefined,
+  };
```

Comment:
The types don't support them but the props still work? That's annoying!

Reply:
Done, although, oddly, once I add it for one prop, the other props stop complaining

Reply:
Hmmm, ok, that didn't work so I'm reverting back to the extraProps... see https://github.com/rjsf-team/react-jsonschema-form/actions/runs/3129403515/jobs/5078449932